config/routes.rb

```diff
@@ -5,6 +5,7 @@
     get  "/help", to: "static_pages#help"
     get  "/about", to: "static_pages#about"
     get  "/contact", to: "static_pages#contact"
-    get  "/signup",to: "user#new"
+    get  "/signup", to: "users#new"
+    resources :users
```

Comment:
nếu đã khai báo get  "/signup" rồi thì chỗ resources này ko cần dùng cho new nữa nha em, dùng cái nào thì khai báo cái đó thôi
```suggestion
    resources :users, only: :show
```